yoga/Yoga.c

```diff
@@ -2339,7 +2339,12 @@ static void YGNodelayoutImpl(const YGNodeRef node,
                                           availableInnerCrossDim) +
                            marginCross;
           childCrossMeasureMode =
-              YGFloatIsUndefined(childCrossSize) ? YGMeasureModeUndefined : YGMeasureModeExactly;
+              YGFloatIsUndefined(childCrossSize) ||
+                      (currentRelativeChild->resolvedDimensions[dim[crossAxis]]->unit ==
```

Comment:
Could we extract this condition into a named bool? Or add comments on the two conditions we're trying to check here?